discord/ext/commands/help.py

```diff
@@ -1341,3 +1342,170 @@ async def send_command_help(self, command):
+            {alias_text}
+            """
+        )
+        await self.context.send(embeds=[em])
+    
+    async def send_cog_help(self, cog):
+        ctx = self.context
+        cog_cmds = cog.get_commands()
+
+        # Sort the commands and groups
+        cmds = ["**Commands:**"]
+        groups = ["**Command Groups:**"]
+        indent = "\u200b " * 4
+        for cmd in cog_cmds:
+            if not isinstance(cmd, Command):
+                continue
+            if cmd.hidden == True:
+                continue
+            if isinstance(cmd, Group):
+                groups.append(
+                    f'{indent}__{cmd.name.title()}__{": " + cmd.short_doc if cmd.short_doc != "" else cmd.short_doc}'
+                )
+            else:
+                cmds.append(
+                    f'{indent}__{cmd.name.title()}__{": " + cmd.short_doc if cmd.short_doc != "" else cmd.short_doc}'
+                )
+        
+        # Create and send the embed
+        em = discord.Embed(
+            title=f'{cog_cmds[0].cog.qualified_name.title()} Command List',
+            description=""
+        )
+        em_desc = []
+        em_desc = cmds + [""] + groups
+        em.description = "\n".join(em_desc)
+        em.description = f'{em.description}\n\n'
+        em.description = f'{em.description}\nType `{ctx.clean_prefix}help <command>` for more info on a command.'
+        em.description = f'{em.description}\nType `{ctx.clean_prefix}help <group>` for more info on a group.'
+        await self.context.send(embeds=[em])
+    
+    async def send_group_help(self, group):
+        ctx = self.context
+        
+        # Sort the subcommands and subgroups
+        cmds = [f'**Subcommands:**']
+        groups = [f'**Subgroups:**']
+        indent = "\u200b " * 4
+        for cmd in group.commands:
+            if isinstance(cmd, Group):
+                groups.append(
+                    f'{indent}__{cmd.name.title()}__{": " + cmd.short_doc if cmd.short_doc != "" else cmd.short_doc}'
+                )
+            else:
+                cmds.append(
+                    f'{indent}__{cmd.name.title()}__{": " + cmd.short_doc if cmd.short_doc != "" else cmd.short_doc}'
+                )
+        if len(cmds) < 2:
+            cmds.append(f"{indent}None")
+        if len(groups) < 2:
+            groups.append(f"{indent}None")
+        
+        # Create and send the embed
+        em = discord.Embed(
+            title=f'__**Command Group: {group.name.title()}**__',
+            description=""
+        )
+        em_desc = []
+        em_desc = cmds + [""] + groups
+        em.description = "\n".join(em_desc)
+        em.description = f'{em.description}\n\n'
+        em.description = f'{em.description}\nType `{ctx.clean_prefix}help {group.qualified_name} <subcommand>` for more info on a subcommand.'
+        em.description = f'{em.description}\nType `{ctx.clean_prefix}help {group.qualified_name} <subgroup>` for more info on a subgroup.'
```

Comment:
Same here, `+=` could be used instead
```suggestion
        em.description += f'\n\n'
        em.description += f'\nType `{ctx.clean_prefix}help {group.qualified_name} <subcommand>` for more info on a subcommand.'
        em.description += f'\nType `{ctx.clean_prefix}help {group.qualified_name} <subgroup>` for more info on a subgroup.'
```